viya4-iac-aws-darksite/README.md

```diff
@@ -0,0 +1,39 @@
+# Using the `viya4-iac-aws` GitHub project in a Dark Site Deployment Scenario (Experimental)
+
+### Contributors
+
+We thank the following individuals for technical assistance and their contributions for the documentation, scripts and yaml templates that provided the basis for this document.
+- Josh Coburn
+- Matthias Ender
+
+### Background for Dark Site Deployment Scenarios
+
+This documentation contains procedures that can be used to successfully complete a Dark Site deployment using elements of `viya4-iac-aws` GitHub project decribed below.
+
+#### Dark Site Deployment Target Recommendation
+**Note:** _It is recommended for a Dark Site deployment that you use a [BYON](https://github.com/sassoftware/viya4-iac-aws/blob/main/docs/user/BYOnetwork.md) or Bring Your Own Network configuration (scenario 3)  with IAC by creating your own VPC, subnets, AWS private endpoints, and network security groups versus letting `viya4-iac-aws` create these items for you.  If you rely on `viya4-iac-aws` to create the network infrastructure in a Dark Site scenario, you will likely run into failure points.  The procedures listed below assume that you use a BYON configuration scenario with IAC._
+
+### Procedures
+
+1.  **Build a Private VPC for EKS:**
+
+    - **Note:** _Creating AWS VPCs for an EKS cluster can be accomplished in many different ways. Some of those methods include using a CloudFormation template, the AWS console, the AWS CLI, or Terraform scripts._
+
+    - **Note**: If you have an existing VPC environment consisting of a private VPC and a public VPC (optional) that constitutes a Dark Site configuration, skip to the **_Create Custom AMI for Jumpserver/NFSServer_** in step #2 below.
+
+    - Background Information - AWS Reference documentation with details on how to create a VPC for any EKS cluster: https://docs.aws.amazon.com/eks/latest/userguide/creating-a-vpc.html
+
+    - For the cluster VPC ranges, `viya4-iac-aws` defaults to using a CIDR of /16. The `viya4-iac-aws` project creates a /21 VPC with both /22 and /28 sized subnets.  Excluding the `control_plane` subnets, the sizes were chosen by estimating the number of pods (doubling that number to account for viya updates), services, AWS overhead, and then adding a few hundred as a buffer.  
```

Comment:
VPC CIDR ranges is not a dark site specific topic, it's a BYON topic, and I think if we are going to start documenting recommendations about CIDR ranges, that should live in the BYON doc.

Reply:
@canpmh, agreed, The VPC CIDR range para has been removed.